semconv/src/main/java/io/opentelemetry/semconv/trace/attributes/SemanticAttributes.java

```diff
@@ -16,9 +16,696 @@
+
+  /** The name of the StatefulSet. */
+  public static final AttributeKey<String> K8S_STATEFULSET_NAME = stringKey("k8s.statefulset.name");
+
+  /** The UID of the DaemonSet. */
+  public static final AttributeKey<String> K8S_DAEMONSET_UID = stringKey("k8s.daemonset.uid");
+
+  /** The name of the DaemonSet. */
+  public static final AttributeKey<String> K8S_DAEMONSET_NAME = stringKey("k8s.daemonset.name");
+
+  /** The UID of the Job. */
+  public static final AttributeKey<String> K8S_JOB_UID = stringKey("k8s.job.uid");
+
+  /** The name of the Job. */
+  public static final AttributeKey<String> K8S_JOB_NAME = stringKey("k8s.job.name");
+
+  /** The UID of the CronJob. */
+  public static final AttributeKey<String> K8S_CRONJOB_UID = stringKey("k8s.cronjob.uid");
+
+  /** The name of the CronJob. */
+  public static final AttributeKey<String> K8S_CRONJOB_NAME = stringKey("k8s.cronjob.name");
+
+  /** The operating system type. */
+  public static final AttributeKey<String> OS_TYPE = stringKey("os.type");
+
+  /**
+   * Human readable (not intended to be parsed) OS version information, like e.g. reported by {@code
+   * ver} or {@code lsb_release -a} commands.
+   */
+  public static final AttributeKey<String> OS_DESCRIPTION = stringKey("os.description");
+
+  /** Human readable operating system name. */
+  public static final AttributeKey<String> OS_NAME = stringKey("os.name");
+
+  /**
+   * The version string of the operating system as defined in <a
+   * href="../../resource/semantic_conventions/README.md#version-attributes">Version Attributes</a>.
+   */
+  public static final AttributeKey<String> OS_VERSION = stringKey("os.version");
+
+  /** Process identifier (PID). */
+  public static final AttributeKey<Long> PROCESS_PID = longKey("process.pid");
+
+  /** Parent Process identifier (PID). */
+  public static final AttributeKey<Long> PROCESS_PARENT_PID = longKey("process.parent_pid");
+
+  /**
+   * The name of the process executable. On Linux based systems, can be set to the {@code Name} in
+   * {@code proc/[pid]/status}. On Windows, can be set to the base name of {@code
+   * GetProcessImageFileNameW}.
+   */
+  public static final AttributeKey<String> PROCESS_EXECUTABLE_NAME =
+      stringKey("process.executable.name");
+
+  /**
+   * The full path to the process executable. On Linux based systems, can be set to the target of
+   * {@code proc/[pid]/exe}. On Windows, can be set to the result of {@code
+   * GetProcessImageFileNameW}.
+   */
+  public static final AttributeKey<String> PROCESS_EXECUTABLE_PATH =
+      stringKey("process.executable.path");
+
+  /**
+   * The command used to launch the process (i.e. the command name). On Linux based systems, can be
+   * set to the zeroth string in {@code proc/[pid]/cmdline}. On Windows, can be set to the first
+   * parameter extracted from {@code GetCommandLineW}.
+   */
+  public static final AttributeKey<String> PROCESS_COMMAND = stringKey("process.command");
```

Comment:
Something doesn't seem right. There's lots of resource attributes in here.

Reply:
hmmm. the generator is not in a great state right now. probably something more needs tweaking. I tried only generating span attributes, but then we ended up with removed attributes that are tagged as type=event. Not sure how to properly tweak the generator at this point, to get what we want, and not get what we don't want.